velox/expression/VectorFunction.cpp

```diff
@@ -70,6 +70,16 @@ std::optional<std::vector<FunctionSignaturePtr>> getVectorFunctionSignatures(
       });
 }
 
+std::optional<
+    std::pair<VectorFunctionMetadata, std::vector<FunctionSignaturePtr>>>
+getVectorFunctionSignaturesAndMetadata(const std::string& name) {
```

Comment:
Maybe return std::optional<std::vector<FunctionSignaturePtr>, 
    std::pair<VectorFunctionMetadata>>, the order is same with function name

Reply:
Updated. Thanks.